juicy-html.html

```diff
@@ -134,7 +186,26 @@
                         this.model = newVal;
                         break;
                     case "content":
-                        this.loadTemplate_();
+                        warnAboutDeprecatedContentOn(this);
+                        forwardContentValue(this, newVal);
+                        break;
+                    case "href":
+                        // skip pending request
+                        this.skipStampingPendingFile();
```

Comment:
This line should be both in case `href` and case `html`.

Reply:
Not really, 
- `element.pending && element.setAttribute('html', 'smth')` - 
     `this.removeAttribute('href')` (=>`this.skipStampingPendingFile();`)
- `element.pending && element.removeAttribute('html')` - 
    why would you skip pending?

especially given that we do not (want to) control the order of set attributes by some data-binding systems:
`<template is="juicy-html" href="{{href}}" html="{{html}}">`, then change `{href: null, html: 'smth'}` to `{href:'foo.html', html:null}`
If the order would be:
```js
setAttribute('href', 'foo.html'); // start a request
removeAttribute('html'); // skip it
```
the loaded content will not get stamped.

Reply:
You are right, essentially setting `html` removes `href`, which calls `this.skipStampingPendingFile();`.

So as long as `html` setter has an unsetter for `href`, this will work well.